en/docs/fragments/attribute-dialects-oidc.md

```diff
@@ -0,0 +1,13 @@
+1.  Navigate to the **Attributes** tab. 
+
+2.  Click the edit icon. 
+
+3.  Select the required local attributes for this application. For this scenario, we have chosen `address`, `country`, `email`, and `name`. 
```

Comment:
Choose --> Select

(generally we've used the word "select" when it comes to dropdown lists)

Reply:
For this scenario, select `address`,....